examples/fabric.rb

```diff
@@ -0,0 +1,31 @@
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+my_client = {
+  url: ENV['ONEVIEWSDK_URL'],
+  user: ENV['ONEVIEWSDK_USER'],
+  password: ENV['ONEVIEWSDK_PASSWORD']
+}
+
+# Does nothing (default :none)
+oneview_fabric 'DefaultFabric' do
+  client my_client
+end
+
+# Updates the DefaultFabric fabric reserved vlan range if the values do not match
+oneview_fabric 'DefaultFabric' do
+  client my_client
+  reserved_vlan_range(
+    'start' => 3000,
+    'length' => 120
+  )
+  action :set_reserved_vlan_range
+end
```

Comment:
@tmiotto , no. It doesn't use the client, but it does use the optional properties `api_version` and `api_variant`; see [here](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/oneview_helper.rb#L33). Those default to the attributes defined by the cookbook; see [here](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/oneview_resource_base.rb#L21) and [here](https://github.com/HewlettPackard/oneview-chef/blob/master/attributes/default.rb#L27).

So based on the example code I see, it would use the cookbook defaults: `api_version=200` & `api_variant='C7000'`